pom.xml

```diff
@@ -59,7 +59,7 @@
         <dep.selenium.version>3.141.59</dep.selenium.version>
         <dep.tempto.version>185</dep.tempto.version>
         <dep.gcs.version>2.0.0</dep.gcs.version>
-        <dep.errorprone.version>2.8.1</dep.errorprone.version>
+        <dep.errorprone.version>2.9.0</dep.errorprone.version>
```

Comment:
Can you add some justification in the commit message? Was this to satisfy some classpath issues when we update BigQuery dependencies?

Reply:
Added a commit description as below. 
> New GCS library requires new errorprone version.

Reply:
Thanks for updating that. I had it on my list :)